suave/builder/session_manager.go

```diff
@@ -150,6 +150,30 @@ func (s *SessionManager) AddTransaction(sessionId string, tx *types.Transaction)
 	return builder.AddTransaction(tx)
 }
 
+func (s *SessionManager) AddTransactions(sessionId string, txs types.Transactions) ([]*api.SimulateTransactionResult, error) {
+	builder, err := s.getSession(sessionId)
+	if err != nil {
+		return nil, err
+	}
+	return builder.AddTransactions(txs)
+}
+
+func (s *SessionManager) AddBundle(sessionId string, bundle *api.Bundle) (*api.SimulateBundleResult, error) {
```

Comment:
Can we merge `AddBundle` with `AddBundles`? That way we can reduce the scope of the API.

Reply:
Note that we cannot do the same with `AddTransactions` because of the backward compatibility.

Reply:
Done